### ISSUE TITLE
2021 Volunteer callout blog post

### ISSUE THREAD

**Issue by funnelfiasco** [SeaGL/organization#55]
Once we've finalized the committee structure, we should put out a post highlighting the work each one does.

**Comment by funnelfiasco** [comment]
Started a [draft](https://github.com/SeaGL/seagl.github.io/blob/volunteers_wanted_post/_posts/2021-07-01-volunteers_wanted.md) in the repo.

**Comment by koronkowy** [comment]
> Started a [draft](https://github.com/SeaGL/seagl.github.io/blob/volunteers_wanted_post/_posts/2021-07-01-volunteers_wanted.md) in the repo. Send an email to staff@ to get input from others.

Did this end up going out? I'd be happy to look at it but I won't have it as I joined after.

I've been asking committees the types of volunteers they need but am happy to also take a stab at some words for this.

**Comment by altsalt** [comment]
> Did this end up going out?

I don't believe so, this was drafted in a branch but stalled here. I'm sure you're welcome to it!

**Comment by funnelfiasco** [comment]
> I've been asking committees the types of volunteers they need but am happy to also take a stab at some words for this.

Be my guest, @koronkowy!
